hack/run-functests.sh

```diff
@@ -1,15 +1,21 @@
 #!/bin/bash
 set -x
-which ginkgo
+# Install Go 1.20.4
+wget https://go.dev/dl/go1.20.4.linux-amd64.tar.gz
+rm -rf /usr/local/go
```

Comment:
So this will remove existing go binary on a user's system?

Reply:
The version of ginkgo used below requires at least this version of golang. This change will enable the CI setup to run the tests as the image used for CI has an older version of golang.. there is an ongoing effort to change the image used after which this change can be rolled back